styleguide/STYLE.md

```diff
@@ -455,8 +455,8 @@ This information was aggregated by going over the existing screenshots in the do
 
 ### Taking and processing screenshots
 * **Format:** PNG
-* **Ratio:** arbitrary, but **strictly even number of pixels** for both height and width. Recommended size `width: 1024px, height: 880px` when you're taking a full-width screen
-* **Size:** the largest possible resolution that makes sense. It's preferable that you take the screenshots on a Mac laptop with a Retina screen using Safari. Images should be exported at double (`@2x`) the original screen. Recommended dimension is `width: 2048/2, height: 880/2` to get the best possible view across different screen sizes.
+* **Ratio:** arbitrary, Recommended size `width: 1024px, height: 880px` when you're taking a full-width screen
```

Comment:
What about the retina screens? 1024 x 880 seems tiny for a full-width screen screenshot, especially for the retina and 2K, 4K screens.

Reply:
Ah good point. I'll tweak this a bit. We should be taking screenshots on an @2x retina screen. So whilst the image is 1024 * 880, the native image size is 2048 * 1760